server/config-docs.ts

```diff
@@ -75,6 +75,19 @@ const validator = ajv.compile({
                     type: "string",
                   },
                 },
+                forScopes: {
+                  anyOf: [
+                    {
+                      type: "string",
+                    },
+                    {
+                      type: "array",
+                      items: {
+                        type: "string",
+                      },
+                    },
+                  ],
+                },
```

Comment:
What is the reasoning to allow array OR string (with all permutations of the options?)

Reply:
It costs almost nothing to process an array. Strings to achieve uniformity.
The array has been added to the processing to avoid errors.